tests/perf/test_result.go

```diff
@@ -70,3 +70,25 @@ type TestResult struct {
 	SocketCount int    `json:"SocketCount"`
 	AbortOn     int    `json:"AbortOn"`
 }
+
+type TestReport struct {
+	Results            []TestResult `json:"Results"`
+	TestName           string       `json:"TestName"`
```

Comment:
Can we group the GitHub stuff and CPU/Memory consumption into separate parents?

Reply:
{
  "TestName": ...,
  "Results": ...,
  "GitHub": ...,
  "TestMetrics": ...,
}

Reply:
Done